src/frontend/templates/pages/Login.js

```diff
@@ -0,0 +1,84 @@
+import React , { useState } from 'react';
+import { useNavigate , Link } from 'react-router-dom';
+import '../../styles/pages/auth.css';
+import { useUser } from '../../contexts/userContext';
+import loginFormInputs from '../forms/LoginFormArray';
+import { loginAction } from '../../services/authService';
+import { toastError, toastSuccess } from '../../services/toastService';
+
+export const Login = () => {
+  
+    const { userDispatcher } = useUser();
+    const navigate = useNavigate();   
+    const [credentials,setCredentials] = useState({ email : '', password : '' });
+    
+    const handleCredentialsChange = (e) => {
+        const { name,value } = e.target;
+        const currentCredentials = { ...credentials , [name]:value }
+        setCredentials(currentCredentials);
+    } 
+    
+    const Login = async (e) => {
+    e.preventDefault();
+    let response;
+    try {
+        response = await loginAction(credentials);
+        if (response.actionSuccess) {
+            let { foundUser , encodedToken } = response.actionResponse.data;
+            toastSuccess('You have Succesfully Logged In!');
+            userDispatcher({ type: 'login' , payload : { foundUser , encodedToken }});
+            navigate('/');
+        } else {
+            if(response.actionResponse.response.status === 404 || response.actionResponse.response.status === 401 ) {
+            toastError(response.actionResponse.response.data.errors[0].split('.')[0]); 
+           } else {
+            toastError('Oops, an error has occured!');
+           }
+        }
+    } catch(error) {
+        response = error;
```

Comment:
please avoid doing this naming it creates a little confusion

Reply:
thanks for this shivam , will try to better my naming convention